### ISSUE TITLE
ML Kit FaceDetectionProcessor from sample app not detecting ears

### ISSUE THREAD

**Issue by MarianIacob** [firebase/quickstart-android#700]
### Issues details:
I tried the ML Kit Face detection sample app from [here](https://github.com/firebase/quickstart-android/blob/master/mlkit/README.md) but was not able to receive landmark data for the ears while running the LiveDataPreviewActivity. The call **_face.getLandmark_** always returns **_null_** for FirebaseVisionFaceLandmark.LEFT_EAR and FirebaseVisionFaceLandmark.RIGHT_EAR. All other landmarks are highlighted and retrieved just fine.
Should this work fine? Or is this issue beta version related?

#### Tested on:
Google Pixel (Android 8.1.0)
Samsung Galaxy S6 (Android 7.0)

#### Relevant Code:
```
drawLandmarkPosition(canvas, face, FirebaseVisionFaceLandmark.LEFT_EAR)
drawLandmarkPosition(canvas, face, FirebaseVisionFaceLandmark.RIGHT_EAR)
...
private fun drawLandmarkPosition(canvas: Canvas, face: FirebaseVisionFace, landmarkID: Int) {
        val landmark = face.getLandmark(landmarkID)
        landmark?.let {
            val point = landmark.position
            canvas.drawCircle(
                    translateX(point.x),
                    translateY(point.y),
                    10f, idPaint)
        }
    }
```

https://github.com/firebase/quickstart-android/blob/master/mlkit/app/src/main/java/com/google/firebase/samples/apps/mlkit/kotlin/facedetection/FaceGraphic.kt#L115

**Comment by MarianIacob** [comment]
Hi all,

I received yesterday an official answer from firebase tech support (thanks again!) and they recommended adding this in build.gradle:
_implementation 'com.google.firebase:firebase-ml-vision-face-model:17.0.2'_
... and that was it. Ears are now detected properly in the sample app ;)

I created a pull request for this... hope it gets accepted :)

Cheers and all the best,
Marian.

**Comment by samtstern** [comment]
Fixed in #709
